instrumentation/aws-sdk/aws-sdk-1.11/src/main/java/io/opentelemetry/auto/instrumentation/awssdk/v1_11/OnErrorTracer.java

```diff
@@ -16,8 +16,13 @@
 
 package io.opentelemetry.auto.instrumentation.awssdk.v1_11;
 
-import io.opentelemetry.auto.bootstrap.instrumentation.decorator.BaseDecorator;
+import io.opentelemetry.auto.bootstrap.instrumentation.decorator.BaseTracer;
 
-public class OnErrorDecorator extends BaseDecorator {
-  public static final OnErrorDecorator DECORATE = new OnErrorDecorator();
+public class OnErrorTracer extends BaseTracer {
+  public static final OnErrorTracer ERROR_TRACER = new OnErrorTracer();
+
+  @Override
+  protected String getInstrumentationName() {
+    return "io.opentelemetry.auto.aws-sdk-1.11-error";
```

Comment:
Not sure if i have followed the right naming convention here.

Reply:
current convention is for this to be based on module name only, e.g. `io.opentelemetry.auto.aws-sdk-1.11`

Reply:
AwsSdkClientTracer has already used "io.opentelemetry.auto.aws-sdk-1.11".

Reply:
it's ok, the SDK will return the same `Tracer` when we re-use the same `instrumentationName` / `instrumentationVersion` pair

Reply:
This is fixed in #922.